profile.html

```diff
@@ -80,14 +80,14 @@ <h4 class="modal-title">Add a new expenditure</h4>
             </div>
           </div>
         </div>
+        <button
+                id="addExpenditureBtn"
+                type="button"
+                class="btn"
+                data-toggle="modal"
+                data-target="#addExpenditureModal"
+        >Add Expenditure</button>
         <div class="button-container">
-          <button 
-            id="addExpenditureBtn" 
```

Comment:
The button should be inside of the button-container, might be another merge conflict incorrectly resolved. If not, can you explain why the button is outside of its container in the new version?